core/templates/domain/exploration/stats-reporting-backend-api.service.ts

```diff
@@ -224,7 +224,15 @@ export class StatsReportingBackendApiService {
       client_time_spent_in_secs: clientTimeSpentInSecs,
       old_state_name: oldStateName,
       answer_group_index: answerGroupIndex,
-      rule_spec_index: ruleSpecIndex,
+      // Rules used to be stored in a list of Rules, but have been moved to a
```

Comment:
(Note that, in contrast to my previous comment, this docstring is OK and does make sense in context.)